pkg/activator/handler/handler.go

```diff
@@ -84,12 +98,25 @@ func (a *ActivationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		Host:   host,
 	}
 
+	// Setup tracing transport if configured
+	tr := a.Transport
+	traceTr, err := zipkinhttp.NewTransport(tracer, zipkinhttp.RoundTripper(tr))
+	if err != nil {
+		a.Logger.Errorw("Failed to create zipkin http transport", zap.Error(err))
+	} else {
+		tr = traceTr
+	}
+
 	err = a.Throttler.Try(revID, func() {
 		var (
 			httpStatus int
 			attempts   int
+			probeSpan  zipkin.Span
+			proxySpan  zipkin.Span
```

Comment:
I don't quite get why we adapted variable hoisting in so many places now. The proxySpan for instance is only needed in the `if success` block below.